eidoscommon/src/main/scala/org/clulab/wm/eidoscommon/EidosProcessor.scala

```diff
@@ -109,6 +113,32 @@ class EidosPortugueseProcessor(val language: String, cutoff: Int) extends Portug
   def getTagSet: TagSet = tagSet
 }
 
+class EidosCluProcessor(val language: String, cutoff: Int) extends FastNLPProcessorWithSemanticRoles
+  with EidosProcessor with SentencesExtractor with LanguageSpecific {
+  lazy val eidosTokenizer: EidosTokenizer = new EidosTokenizer(localTokenizer, cutoff)
+  override lazy val tokenizer = new EidosTokenizer(localTokenizer, cutoff)
+  val tagSet = new EnglishTagSet()
+  val cluproc = new CluProcessor()
```

Comment:
I don't see that cluproc is ever used, so I don't understand why the code is there.  Is it stale?

Reply:
It's used when we want to have access to semantic roles

Reply:
but enabled through the config, so it may not be greppable easily in the code

Reply:
IntelliJ's "Find Usages" is apparently blind to it.  Perhaps the use is outside its field of vision.  The `FastNLPProcessorWithSemanticRoles` has a `CluProcessor` itself, so I think one can do a
```scala
  val cluproc = this.cluProcessor
```
or just use `cluProcessor` instead of `cluproc`.  The things are fairly large and take time to load.

Reply:
I think Keith is correct here. I tried running the debug grounder without `cluproc` (and without the line initializing dynet), and it still seems to work, giving me semantic roles and everything.